AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Actions/Persistence/ConfigureAuthentication.swift

```diff
@@ -24,7 +24,7 @@ struct ConfigureAuthentication: Action {
             let signedInData = SignedInData(userId: "",
                                             userName: "",
                                             signedInDate: Date(),
-                                            signInMethod: .srp,
+                                            signInMethod: .apiBased(.userSRP),
```

Comment:
Would we need to store this value in the keychain too?

Reply:
we might, I think we can handle it when we add other signIn mechanisms.